addons/mail/security/ir.model.access.csv

```diff
@@ -1,48 +1,58 @@
-access_mail_alias_all,mail.alias.all,model_mail_alias,,1,0,0,0
-access_mail_alias_user,mail.alias.user,model_mail_alias,base.group_user,1,0,0,0
-access_mail_alias_system,mail.alias.system,model_mail_alias,base.group_system,1,1,1,1
-access_mail_message_subtype_all,mail.message.subtype.all,model_mail_message_subtype,,1,0,0,0
-access_mail_message_subtype_user,mail.message.subtype.user,model_mail_message_subtype,base.group_user,1,1,1,1
-access_mail_tracking_value_all,mail.tracking.value.all,model_mail_tracking_value,,0,0,0,0
-access_mail_tracking_value_portal,mail.tracking.value.portal,model_mail_tracking_value,base.group_portal,0,0,0,0
-access_mail_tracking_value_user,mail.tracking.value.user,model_mail_tracking_value,base.group_user,0,0,0,0
-access_mail_tracking_value_system,mail.tracking.value.system,model_mail_tracking_value,base.group_system,1,1,1,1
-access_publisher_warranty_contract_all,publisher.warranty.contract.all,model_publisher_warranty_contract,,1,1,1,1
-access_mail_template,mail.template,model_mail_template,base.group_user,1,1,1,0
-access_mail_template_system,mail.template_system,model_mail_template,base.group_system,1,1,1,1
-access_mail_shortcode,mail.shortcode,model_mail_shortcode,base.group_user,1,1,1,1
-access_mail_shortcode_portal,mail.shortcode.portal,model_mail_shortcode,base.group_portal,1,0,0,0
-access_mail_activity_all,mail.activity.all,model_mail_activity,,0,0,0,0
-access_mail_activity_user,mail.activity.user,model_mail_activity,base.group_user,1,1,1,1
-access_mail_activity_type_all,mail.activity.type.all,model_mail_activity_type,,0,0,0,0
-access_mail_activity_type_user,mail.activity.type.user,model_mail_activity_type,base.group_user,1,0,0,0
-access_mail_activity_type_system,mail.activity.type.system,model_mail_activity_type,base.group_system,1,1,1,1
-access_mail_blacklist_system,access_mail_blacklist_system,model_mail_blacklist,base.group_system,1,1,1,1
-access_mail_wizard_invite,access.mail.wizard.invite,model_mail_wizard_invite,base.group_user,1,1,1,0
-access_mail_compose_message,access.mail.compose.message,model_mail_compose_message,base.group_user,1,1,1,0
-access_mail_compose_message_portal,access.mail.compose.message.portal,model_mail_compose_message,base.group_portal,1,1,1,0
-access_mail_resend_cancel,access.mail.resend.cancel,model_mail_resend_cancel,base.group_user,1,1,1,0
-access_mail_resend_message,access.mail.resend.message,model_mail_resend_message,base.group_user,1,1,1,0
-access_mail_resend_partner,access.mail.resend.partner,model_mail_resend_partner,base.group_user,1,1,1,0
-access_mail_template_preview,access.mail.template.preview,model_mail_template_preview,base.group_user,1,1,1,0
-access_mail_blacklist_remove_system,acesss.mail.blacklist.remove.system,model_mail_blacklist_remove,base.group_system,1,1,1,1
-ir_actions_report_access_user,ir.actions.report.access.user,base.model_ir_actions_report,base.group_user,1,0,0,0
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
@mart-e I think it's a bug with github diffing with multiple commits, eols are all LF.

Reply:
> eols are all LF.

Which is exactly the issue, before this change they're crlf.

Reply:
Yep, please keep a minimal diff. Especially with ACLs, especially in mail.

Reply:
Indeed I've made the remarks 3 times already during dev, I guess this file doesn't use standard end line and someone must have an auto convert (to the supposedly good format, but it's not desirable here).

Reply:
>  I guess this file doesn't use standard end line

That's correct, some of the older files have windows-style ending instead of unix-style. That's especially prevalent amongst ir.model.access.csv files (there are at least 30 in this case), likely because excel, or because some of the older ones were created windows-style then got copied to create newer ones, and the editors (mostly correctly) kept using the existing line-ending style automatically.

Reply:
Must be from my very short period trying to use WSL on my computer at home, I have stopped now, promise :D